### PR TITLE
Support RETURN AS in SEARCH

### PR DESCRIPTION
**Pull request by AvitalFineRedis**

### CODE REVIEW

**Comment by AvitalFineRedis** [comment]
@chayim Why the codecov says I didn't cover these methods?

File: Makefile [hunk 1]
```diff
@@ -60,3 +60,8 @@ godoc:
 	echo "Open browser tab on localhost:6060"
 	$(GODOC)
 
+start-redis:
+	@docker run --name redisearch-go-tests -d --rm -p 6379:6379 redislabs/redisearch:edge
```
Comment (chayim) [inline Makefile:64]: The name of the docker should be in a variable.

File: Makefile [hunk 2]
```diff
@@ -60,3 +60,8 @@ godoc:
...
+
+stop-redis:
+	@docker stop redisearch-go-tests
```
Comment (chayim) [inline Makefile:67]: Dockername in variable

File: redisearch/query.go [hunk 3]
```diff
@@ -297,6 +297,20 @@ func (q *Query) SetReturnFields(fields ...string) *Query {
 	return q
 }
 
+// AddReturnFields adds the fields that should be returned from each result
+// to the ReturnFields property
+func (q *Query) AddReturnFields(fields ...string) *Query {
```
Comment (chayim) [inline redisearch/query.go:302]: Take a look at this for great doc foo. https://github.com/googleapis/googleapis/blob/master/google/api/http.proto

**Review by chayim — CHANGES_REQUESTED** [review]
Tiny changes requested.

**Review by chayim — APPROVED** [review]

**Pull request merged**